HPCC-14549 RowTag required for XML Spray

g-pan opened the pull request:
Fix HPCC-14549 RowTag required for XML Spray
XML Spray module included in multiple books
This change will update all books referencing the XML Spray module.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>
@JamesDeFabia please review.

g-pan committed:
HPCC-14549 RowTag required for XML Spray

Fix HPCC-14549 RowTag required for XML Spray
XML Spray module included in multiple books
This change will update all books referencing the XML Spray module.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

JamesDeFabia reviewed (changes requested):
The same  change needs to be made for command line DFUPlus and Standard Library  STD.File.SprayXML

HPCCSmoketest commented:
Automated Smoketest
Sha: 71f60a00ccc98b66257e79eff251f0fab7193b7c
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 92.258 seconds |

Install hpccsystems-platform-community_6.2.0-rc5.el7.x86_64.rpm
HPCC Start: OK

Unittest result:

| total | passed | failed | timeout | 
|---|---|---|---|
|88 |88 |0 |0 |

HPCC Stop: OK
HPCC Uninstall: OK

g-pan committed:
HPCC-14549 RowTag required for XML Spray

Fix HPCC-14549 RowTag required for XML Spray
XML Spray module included in multiple books
This change will update all books referencing the XML Spray module.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

JamesDeFabia reviewed (approved):
Looks good to merge

HPCCSmoketest commented:
Automated Smoketest
Sha: 141a3ca93655ec503382edec8b8c59a421b5c4cc
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 93.386 seconds |

Install hpccsystems-platform-community_6.2.0-rc5.el7.x86_64.rpm
HPCC Start: OK

Unittest result:

| total | passed | failed | timeout | 
|---|---|---|---|
|88 |88 |0 |0 |

HPCC Stop: OK
HPCC Uninstall: OK
